contribs/gmf/src/directives/mobilemeasurelength.js

```diff
@@ -0,0 +1,270 @@
+goog.provide('gmf.MobileMeasureLengthController');
+goog.provide('gmf.mobileMeasureLengthDirective');
+
+goog.require('gmf');
+goog.require('ngeo.interaction.MeasureLengthMobile');
+goog.require('ngeo.interaction.MobileDraw');
+goog.require('ol.style.Fill');
+goog.require('ol.style.RegularShape');
+goog.require('ol.style.Stroke');
+goog.require('ol.style.Style');
+
+
+gmf.module.value('gmfMobileMeasureLengthTemplateUrl',
+    /**
+     * @param {angular.JQLite} element Element.
+     * @param {angular.Attributes} attrs Attributes.
+     */
+    function(element, attrs) {
+      var templateUrl = attrs['gmfMobileMeasureLengthTemplateurl'];
+      return templateUrl !== undefined ? templateUrl :
+          gmf.baseTemplateUrl + '/mobilemeasurelength.html';
+    });
+
+
+/**
+ * Provide a "mobile measure" directive.
+ *
+ * @example
+ * <div gmf-mobile-measure-length=""
+ *   gmf-mobile-measure-length-active="ctrl.measureLengthActive"
+ *   gmf-mobile-measure-length-map="::ctrl.map">
+ * </div>
+ *
+ * @param {string|function(!angular.JQLite=, !angular.Attributes=)}
+ *     gmfMobileMeasureLengthTemplateUrl Template URL for the directive.
+ * @return {angular.Directive} The Directive Definition Object.
+ * @ngInject
+ * @ngdoc directive
+ * @ngname gmfMobileMeasureLength
+ */
+gmf.mobileMeasureLengthDirective =
+    function(gmfMobileMeasureLengthTemplateUrl) {
+  return {
+    restrict: 'A',
+    scope: {
+      'active': '=gmfMobileMeasureLengthActive',
+      'decimals': '=?gmfMobileMeasureLengthDecimals',
```

Comment:
don't do a 2 way binding for decimals
either use `'@'` or get the value from `$attrs`